### PR TITLE
make check for duplicates optional

### PR DESCRIPTION
**Pull request by cg-laser**

### CODE REVIEW

**Comment by cg-laser** [comment]
@christophwelling ok with you?

File: NuRadioReco/modules/triggerTimeAdjuster.py [hunk 1]
```diff
@@ -50,6 +50,9 @@ def run(self, event, station, detector):
             trigger_time = trigger.get_trigger_time()
             for channel in station.iter_channels():
                 trigger_time_channel = trigger_time - channel.get_trace_start_time()
+                print(f"trace start time {channel.get_trace_start_time():.1f}")
```
Comment (anelles) [inline NuRadioReco/modules/triggerTimeAdjuster.py]: Please change all the print statements to logging.

**Comment by christophwelling** [comment]
It's parctically random which event will be returned by the get_event function and having event-dependent changes to the detector description may not work. Other than that, it should work, I think

**Comment by cg-laser** [comment]
the NuRadioRecoio now issues a warning if an run/event id pair is requested that exists more than once.

**Pull request merged**